packages/apps/job-launcher/server/src/modules/manifest/manifest.service.spec.ts

```diff
@@ -0,0 +1,795 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import { faker } from '@faker-js/faker';
+import { createMock } from '@golevelup/ts-jest';
+import { Encryption } from '@human-protocol/sdk';
+import { HttpStatus } from '@nestjs/common';
+import { Test } from '@nestjs/testing';
+import { ethers } from 'ethers';
+import { AuthConfigService } from '../../common/config/auth-config.service';
+import { CvatConfigService } from '../../common/config/cvat-config.service';
+import { PGPConfigService } from '../../common/config/pgp-config.service';
+import { Web3ConfigService } from '../../common/config/web3-config.service';
+import { ErrorJob } from '../../common/constants/errors';
+import {
+  JobCaptchaMode,
+  JobCaptchaRequestType,
+  JobCaptchaShapeType,
+  JobRequestType,
+} from '../../common/enums/job';
+import { ControlledError } from '../../common/errors/controlled';
+import { listObjectsInBucket } from '../../common/utils/storage';
+import { StorageService } from '../storage/storage.service';
+import { Web3Service } from '../web3/web3.service';
+import {
+  createJobAudinoDto,
+  createJobCaptchaDto,
+  createJobCvatDto,
+  getMockedProvider,
+  getMockedRegion,
+  mockAuthConfigService,
+  mockCvatConfigService,
+  mockWeb3ConfigService,
+} from './fixtures';
+import { FortuneManifestDto } from './manifest.dto';
+import { ManifestService } from './manifest.service';
+
+jest.mock('../../common/utils/storage', () => ({
```

Comment:
Please move it to very to of this file: jest automatically hoists `jest.mock` statements, so when they are on top it's easy to see and it's explicit for everyone